frontend/src/utils/format_column_data.ts

```
@@ -14,6 +14,7 @@ export class GraphData {
   drawable_shapes: DrawableShape[] = [];
   canvas_container: HTMLCanvasElement;
   current_radius: number = 0;
+  drawable_shapes_root: number = 0;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)